src/api_client.py

```
@@ -0,0 +1,120 @@
+    def __init__(
+        self,
+        api_key: Optional[str] = None,
+        model: Optional[str] = None,
+        max_tokens: Optional[int] = None,
+        temperature: Optional[float] = None,
+    ):
+        """
+        KansaiTalkerClientの初期化
+
+        Args:
+            api_key: Anthropic APIキー（省略時はConfig.ANTHROPIC_API_KEYを使用）
```

Ruff (0.14.4)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)